.eslintrc.js

```diff
@@ -1,15 +1,141 @@
+// turn off the rule completely
```

Comment:
I have added a `.eslintrc.js` which does not break the existing code in repo. If this settings comes to master there could be many merge conflicts for the existing branches. So, I highly recommend sticking with the exisisting one